python/lsst/daf/butler/cli/cmd/commands.py

```diff
@@ -599,3 +599,31 @@ def ingest_files(**kwargs):
     raw camera exposures.
     """
     script.ingest_files(**kwargs)
+
+
+@click.command(cls=ButlerCommand)
+@repo_argument(required=True)
+@click.argument("dataset_type", required=True)
+@click.argument("storage_class", required=True, nargs=1)
+@click.argument("dimensions", required=False, nargs=-1)
+@click.option("--is-calibration/--no-is-calibration", is_flag=True, default=False,
```

Comment:
"no" instead of "not?

Reply:
The convention is `--x` to enable and `--no-x` to disable and that seems to be what click uses elsewhere. I think I could leave off the negating option but then you'd have no way to set it explicitly rather than letting it be the default.